Lib/tarfile.py

```diff
@@ -46,6 +46,7 @@
 import struct
 import copy
 import re
+import zlib
```

Comment:
`zlib` might be unavailable, you must not import it on module-level. See `_Stream.__init__`.